### ISSUE TITLE
Question: Support for IIIF Image API services

### ISSUE THREAD

**Issue by radpet** [ruven/iipsrv#79]
Hello, I looked into https://github.com/ruven/iipsrv/blob/master/src/IIIF.cc and did not find where/how the service property is passed to the info.json response. I know this is optional property but I suppose it would be great if it is supported.

**Comment by ruven** [comment]
Yes, I guess this could be added. Which service properties are you interested in supporting? I presume things like the [physical dimensions service property](http://iiif.io/api/annex/services/#physical-dimensions)? Resolution info is often included in the image metadata.

But you can also use the service property to point to an authentication method or other info. In that case, this information would need to be embedded within the image metadata, which may not be the ideal way to pass this kind of information.

**Comment by radpet** [comment]
Yup the "physical dimensions service property" but I used the Presentation API instead and have a service which gives me the information.

**Comment by ruven** [comment]
The physical dimensions service property was added in this commit: https://github.com/ruven/iipsrv/commit/7dcd339957cf1073ff28db50b906beaf2174532c
